### ISSUE TITLE
Project management: XOWA TODO list

### ISSUE THREAD

**Issue by gnosygnu** [gnosygnu/xowa#422]
I'm trying this as an experiment. I want to:
* List major open items in one place
* Publicize current priority order
* Allow others to bump up or add new items.

The Primary Issue description will have an evolving list of items. Feel free to comment below

priority | effort | item | issue
-|-|-|-
1-critical | 3-week | Release 2019-03 English Wikipedia
2-major | 3-week | Optimize build for enwiki portal pages | #413 
1-critical | 3-week | Fix issues related to 4.6 release | #404 
4-minor | 4-day | Make XOWA maven friendly
2-major |  4- day | Update Namespace Aliases | #400 
2-major | 2-month | Make VNU default tidy engine | #417
2-major | 2-month | Implement Gadgets |  #420, #329 
more to come ... |

**Comment by desb42** [comment]
When you come to do the wikidata work I would suggest #354 be part of it

**Comment by gnosygnu** [comment]
> When you come to do the wikidata work I would suggest #354 be part of it

Yeah, sorry for missing that. I'll work on it tomorrow morning, and push out something. Thanks.

Also, I reordered some of the priorities above
* Optimize build for enwiki portal pages is now above Release 2019-03 English Wikipedia. There are a bunch of performance issues related to Luaj, and I'd like to try to get that done first. As it stands, portal pages are not usable
* Bumped up a Maven item. I had someone reach out to me offline about a script to make XOWA work via Maven (similar to ant). I'll put up an issue for it tomorrow.

Thanks.

**Comment by gnosygnu** [comment]
Finally knocked off #413 off the list. Working on build for 2019-03 English Wikipedia. Building now and hoping to have released within a week

**Comment by gnosygnu** [comment]
As an alternative to this SIM, I'm going to try the Github Kanban boards: https://github.com/gnosygnu/xowa/projects/1 . If that goes well, then I'll move the issues above to the board and close this issue out
